### PR TITLE
Dawn: Warn against turning off robot during Runtime update

### PR DESCRIPTION
**Pull request by kevinmasd**
Handles #266

### CODE REVIEW

File: dawn/renderer/components/UpdateBox.js [hunk 1]
```diff
@@ -35,12 +37,10 @@ class UpdateBox extends React.Component {
...
       this.setState({ isUploading: false });
-      dialog.showMessageBox({
-        type: 'warning',
-        buttons: ['Close'],
-        title: 'File Issue',
-        message: 'Update File Bad\n',
-      });
+      this.props.onAlertAdd(
+        'File Issue',
+        'Update File Bad',
```
Comment (kevinji) [inline dawn/renderer/components/UpdateBox.js]: Could we give this a more descriptive title/message, like "Issue with file upload" and "There was an issue uploading the current file."?

File: dawn/renderer/components/UpdateBox.js [hunk 2]
```diff
@@ -56,23 +56,19 @@ class UpdateBox extends React.Component {
...
                 this.props.hide();
-                dialog.showMessageBox({
-                  type: 'warning',
-                  buttons: ['Close'],
-                  title: 'Upload Issue',
-                  message: 'Update File Upload Failed.',
-                });
+                this.props.onAlertAdd(
+                  'Upload Issue',
+                  'Could not send Runtime update',
```
Comment (kevinji) [inline dawn/renderer/components/UpdateBox.js]: Ditto.

File: dawn/renderer/components/UpdateBox.js [hunk 3]
```diff
@@ -56,23 +56,19 @@ class UpdateBox extends React.Component {
...
                     if (uperr) {
-                      dialog.showMessageBox({
-                        type: 'warning',
-                        buttons: ['Close'],
-                        title: 'Upload Issue',
-                        message: 'Running Update Script Failed.',
-                      });
+                      this.props.onAlertAdd(
+                        'Upload Issue',
+                        'Update Script Failed',
```
Comment (kevinji) [inline dawn/renderer/components/UpdateBox.js]: Ditto.

**Review by kevinji — DISMISSED** [review]
Mainly just wording issues.

File: dawn/renderer/components/UpdateBox.js [hunk 4]
```diff
@@ -56,23 +48,21 @@ class UpdateBox extends React.Component {
...
                     if (uperr) {
-                      dialog.showMessageBox({
-                        type: 'warning',
-                        buttons: ['Close'],
-                        title: 'Upload Issue',
-                        message: 'Running Update Script Failed.',
-                      });
+                      this.props.onAlertAdd(
+                        'Update Script Error',
+                        `Dawn was unable to run scripts necessary for the update.
```
Comment (yizow) [inline dawn/renderer/components/UpdateBox.js]: Dawn was unable to run update scripts
Reads a bit easier

**Comment by yizow** [comment]
Looks like there are linter errors

**Comment by kevinmasd** [comment]
It's a phantom linter error; lints fine on local machine.

**Comment by kevinmasd** [comment]
Now it's a runtime error

**Review by yizow — APPROVED** [review]

**Pull request merged**